hack/start-ec-task.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/bash
```

Comment:
Should the resources defined here be in infra-deployments?

Reply:
Maybe I'll mention that as a comment.